src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -71,12 +72,39 @@ class NetworkOPsImp final
     : public NetworkOPs
     , public beast::DeadlineTimer::Listener
 {
-public:
-    enum Fault
+    /**
+     * Transaction with input flags and results to be applied in batches.
+     */
+    class TransactionStatus
+    {
+    public:
+        Transaction::pointer transaction;
+        bool admin;
```

Comment:
DANGER WILL ROBINSON!  ;-)

Having two booleans together as parameters is a gun aimed at your foot - because it's so very easy to transpose the two parameters.  I have seen this error over and over again in production code, and I have made this myself.

Strongly prefer an enum class.  The change in readability at the call site is striking.